22:06:34.970 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/zonda_async_ob_update.py", line 27, in main
    '{str(dict_val['message']['changes'][0]['marketCode'])}',
KeyError: 'message'
22:06:35.071 INFO  Ob received for BTC-PLN timestamp: 1642021596605
22:06:35.218 INFO  Ob received for BTC-PLN timestamp: 1642021596696
22:06:35.234 INFO  Ob received for BTC-PLN timestamp: 1642021596763
22:06:35.271 INFO  Ob received for BTC-PLN timestamp: 1642021596810
22:06:35.287 INFO  Ob received for BTC-PLN timestamp: 1642021596812
22:06:35.287 INFO  Ob received for BTC-PLN timestamp: 1642021596821
22:06:35.318 INFO  Ob received for BTC-PLN timestamp: 1642021596829
22:06:35.318 INFO  Ob received for BTC-PLN timestamp: 1642021596841
22:06:35.365 INFO  Ob received for BTC-PLN timestamp: 1642021596892
22:06:35.371 INFO  Ob received for BTC-PLN timestamp: 1642021596895
22:06:35.418 INFO  Ob received for BTC-PLN timestamp: 1642021596947
22:06:35.434 INFO  Ob received for BTC-PLN timestamp: 1642021596948
22:06:35.434 INFO  Ob received for BTC-PLN timestamp: 1642021596965
22:06:35.465 INFO  Ob received for BTC-PLN timestamp: 1642021596982
22:06:35.472 INFO  Ob received for BTC-PLN timestamp: 1642021596995
22:06:35.519 INFO  Ob received for BTC-PLN timestamp: 1642021597013
22:06:35.635 INFO  Ob received for BTC-PLN timestamp: 1642021597174
22:06:35.751 INFO  Ob received for BTC-PLN timestamp: 1642021597271
22:06:35.819 INFO  Ob received for BTC-PLN timestamp: 1642021597340
22:06:35.819 INFO  Ob received for BTC-PLN timestamp: 1642021597358
22:06:35.920 INFO  Ob received for BTC-PLN timestamp: 1642021597454
22:06:36.067 INFO  Ob received for BTC-PLN timestamp: 1642021597574
22:06:36.251 INFO  Ob received for BTC-PLN timestamp: 1642021597786
22:06:36.251 INFO  Ob received for BTC-PLN timestamp: 1642021597798
22:06:36.273 INFO  Ob received for BTC-PLN timestamp: 1642021597801
22:06:36.305 INFO  Ob received for BTC-PLN timestamp: 1642021597820
22:06:36.389 INFO  Ob received for BTC-PLN timestamp: 1642021597877
22:06:36.389 INFO  Ob received for BTC-PLN timestamp: 1642021597878
22:06:36.389 INFO  Ob received for BTC-PLN timestamp: 1642021597881
22:06:36.436 INFO  Ob received for BTC-PLN timestamp: 1642021597972
22:06:36.436 INFO  Ob received for BTC-PLN timestamp: 1642021597975
22:06:36.505 INFO  Ob received for BTC-PLN timestamp: 1642021598008
22:06:36.573 INFO  Ob received for BTC-PLN timestamp: 1642021598120
22:06:36.674 INFO  Ob received for BTC-PLN timestamp: 1642021598205
22:06:36.767 INFO  Ob received for BTC-PLN timestamp: 1642021598309
22:06:36.789 INFO  Ob received for BTC-PLN timestamp: 1642021598327
22:06:36.821 INFO  Ob received for BTC-PLN timestamp: 1642021598354
22:06:36.874 INFO  Ob received for BTC-PLN timestamp: 1642021598409
22:06:36.890 INFO  Ob received for BTC-PLN timestamp: 1642021598429
22:06:36.937 INFO  Ob received for BTC-PLN timestamp: 1642021598439
22:06:36.968 INFO  Ob received for BTC-PLN timestamp: 1642021598498
22:06:37.006 INFO  Ob received for BTC-PLN timestamp: 1642021598527
22:06:37.234 INFO  Ob received for BTC-PLN timestamp: 1642021598700
22:06:37.322 INFO  Ob received for BTC-PLN timestamp: 1642021598861
22:06:37.322 INFO  Ob received for BTC-PLN timestamp: 1642021598869
22:06:37.375 INFO  Ob received for BTC-PLN timestamp: 1642021598875
22:06:37.391 INFO  Ob received for BTC-PLN timestamp: 1642021598887
22:06:37.407 INFO  Ob received for BTC-PLN timestamp: 1642021598894
22:06:37.423 INFO  Ob received for BTC-PLN timestamp: 1642021598947
22:06:37.523 INFO  Ob received for BTC-PLN timestamp: 1642021599028
22:06:37.554 INFO  Ob received for BTC-PLN timestamp: 1642021599048
22:06:37.554 INFO  Ob received for BTC-PLN timestamp: 1642021599088
22:06:37.639 INFO  Ob received for BTC-PLN timestamp: 1642021599176
22:06:37.708 INFO  Ob received for BTC-PLN timestamp: 1642021599219
22:06:37.708 INFO  Ob received for BTC-PLN timestamp: 1642021599220
22:06:37.708 INFO  Ob received for BTC-PLN timestamp: 1642021599228
22:06:37.739 INFO  Ob received for BTC-PLN timestamp: 1642021599238
22:06:37.770 INFO  Ob received for BTC-PLN timestamp: 1642021599305
22:06:37.777 INFO  Ob received for BTC-PLN timestamp: 1642021599315
22:06:37.855 INFO  Ob received for BTC-PLN timestamp: 1642021599397
22:06:37.893 INFO  Ob received for BTC-PLN timestamp: 1642021599416
22:06:37.955 INFO  Ob received for BTC-PLN timestamp: 1642021599493
22:06:38.077 INFO  Ob received for BTC-PLN timestamp: 1642021599603
22:06:38.178 INFO  Ob received for BTC-PLN timestamp: 1642021599712
22:06:38.193 INFO  Ob received for BTC-PLN timestamp: 1642021599725
22:06:38.225 INFO  Ob received for BTC-PLN timestamp: 1642021599769
22:06:38.225 INFO  Ob received for BTC-PLN timestamp: 1642021599772
22:06:38.240 INFO  Ob received for BTC-PLN timestamp: 1642021599782
22:06:38.240 INFO  Ob received for BTC-PLN timestamp: 1642021599784
22:06:38.256 INFO  Ob received for BTC-PLN timestamp: 1642021599798
22:06:38.256 INFO  Ob received for BTC-PLN timestamp: 1642021599803
22:06:38.309 INFO  Ob received for BTC-PLN timestamp: 1642021599846
22:06:38.356 INFO  Ob received for BTC-PLN timestamp: 1642021599870
22:06:38.356 INFO  Ob received for BTC-PLN timestamp: 1642021599893
22:06:38.372 INFO  Ob received for BTC-PLN timestamp: 1642021599910
22:06:38.425 INFO  Ob received for BTC-PLN timestamp: 1642021599950
22:06:38.441 INFO  Ob received for BTC-PLN timestamp: 1642021599953
22:06:38.541 INFO  Ob received for BTC-PLN timestamp: 1642021599980
22:06:38.541 INFO  Ob received for BTC-PLN timestamp: 1642021600049
22:06:38.557 INFO  Ob received for BTC-PLN timestamp: 1642021600085
22:06:38.578 INFO  Ob received for BTC-PLN timestamp: 1642021600113
22:06:38.694 INFO  Ob received for BTC-PLN timestamp: 1642021600204
22:06:38.826 INFO  Ob received for BTC-PLN timestamp: 1642021600363
22:06:38.842 INFO  Ob received for BTC-PLN timestamp: 1642021600381
22:06:38.958 INFO  Ob received for BTC-PLN timestamp: 1642021600483
22:06:39.237 INFO  Ob received for BTC-PLN timestamp: 1642021600746
22:06:39.237 INFO  Ob received for BTC-PLN timestamp: 1642021600747
22:06:39.237 INFO  Ob received for BTC-PLN timestamp: 1642021600749
22:06:39.237 INFO  Ob received for BTC-PLN timestamp: 1642021600760
22:06:39.237 INFO  Ob received for BTC-PLN timestamp: 1642021600761
22:06:39.280 INFO  Ob received for BTC-PLN timestamp: 1642021600823
22:06:39.374 INFO  Ob received for BTC-PLN timestamp: 1642021600877
22:06:39.480 INFO  Ob received for BTC-PLN timestamp: 1642021601003
22:06:39.717 INFO  Ob received for BTC-PLN timestamp: 1642021601255
22:06:39.828 INFO  Ob received for BTC-PLN timestamp: 1642021601351
22:06:39.828 INFO  Ob received for BTC-PLN timestamp: 1642021601356
22:06:39.859 INFO  Ob received for BTC-PLN timestamp: 1642021601376
22:06:39.859 INFO  Ob received for BTC-PLN timestamp: 1642021601390
22:06:39.912 INFO  Ob received for BTC-PLN timestamp: 1642021601446
22:06:40.012 INFO  Ob received for BTC-PLN timestamp: 1642021601456
22:06:40.075 INFO  Ob received for BTC-PLN timestamp: 1642021601593
22:06:40.081 INFO  Ob received for BTC-PLN timestamp: 1642021601601
22:06:40.081 INFO  Ob received for BTC-PLN timestamp: 1642021601605
22:06:40.081 INFO  Ob received for BTC-PLN timestamp: 1642021601613
22:06:40.159 INFO  Ob received for BTC-PLN timestamp: 1642021601688
22:06:40.159 INFO  Ob received for BTC-PLN timestamp: 1642021601696
22:06:40.181 INFO  Ob received for BTC-PLN timestamp: 1642021601719
22:06:40.197 INFO  Ob received for BTC-PLN timestamp: 1642021601723
22:06:40.197 INFO  Ob received for BTC-PLN timestamp: 1642021601733
22:06:40.213 INFO  Ob received for BTC-PLN timestamp: 1642021601738
22:06:40.397 INFO  Ob received for BTC-PLN timestamp: 1642021601804
22:06:40.397 INFO  Ob received for BTC-PLN timestamp: 1642021601908
22:06:40.397 INFO  Ob received for BTC-PLN timestamp: 1642021601914
22:06:40.397 INFO  Ob received for BTC-PLN timestamp: 1642021601926
22:06:40.460 INFO  Ob received for BTC-PLN timestamp: 1642021602005
22:06:40.576 INFO  Ob received for BTC-PLN timestamp: 1642021602104
22:06:40.582 INFO  Ob received for BTC-PLN timestamp: 1642021602127
22:06:40.613 INFO  Ob received for BTC-PLN timestamp: 1642021602140
22:06:40.682 INFO  Ob received for BTC-PLN timestamp: 1642021602208
22:06:40.682 INFO  Ob received for BTC-PLN timestamp: 1642021602220
22:06:40.783 INFO  Ob received for BTC-PLN timestamp: 1642021602289
22:06:40.783 INFO  Ob received for BTC-PLN timestamp: 1642021602296
22:06:40.783 INFO  Ob received for BTC-PLN timestamp: 1642021602305
22:06:40.830 INFO  Ob received for BTC-PLN timestamp: 1642021602363
22:06:40.883 INFO  Ob received for BTC-PLN timestamp: 1642021602417
22:06:40.899 INFO  Ob received for BTC-PLN timestamp: 1642021602431
22:06:40.930 INFO  Ob received for BTC-PLN timestamp: 1642021602479
22:06:40.961 INFO  Ob received for BTC-PLN timestamp: 1642021602503
22:06:41.145 INFO  Ob received for BTC-PLN timestamp: 1642021602588
22:06:41.199 INFO  Ob received for BTC-PLN timestamp: 1642021602718
22:06:41.199 INFO  Ob received for BTC-PLN timestamp: 1642021602750
22:06:41.230 INFO  Ob received for BTC-PLN timestamp: 1642021602751
22:06:41.261 INFO  Ob received for BTC-PLN timestamp: 1642021602757
22:06:41.283 INFO  Ob received for BTC-PLN timestamp: 1642021602766
22:06:41.283 INFO  Ob received for BTC-PLN timestamp: 1642021602767
22:06:41.299 INFO  Ob received for BTC-PLN timestamp: 1642021602768
22:06:41.299 INFO  Ob received for BTC-PLN timestamp: 1642021602772
22:06:41.346 INFO  Ob received for BTC-PLN timestamp: 1642021602873
22:06:41.346 INFO  Ob received for BTC-PLN timestamp: 1642021602874
22:06:41.399 INFO  Ob received for BTC-PLN timestamp: 1642021602942
22:06:41.446 INFO  Ob received for BTC-PLN timestamp: 1642021602979
22:06:41.499 INFO  Ob received for BTC-PLN timestamp: 1642021602980
22:06:41.530 INFO  Ob received for BTC-PLN timestamp: 1642021603055
22:06:41.615 INFO  Ob received for BTC-PLN timestamp: 1642021603150
22:06:41.731 INFO  Ob received for BTC-PLN timestamp: 1642021603250
22:06:41.831 INFO  Ob received for BTC-PLN timestamp: 1642021603353
22:06:41.885 INFO  Ob received for BTC-PLN timestamp: 1642021603422
21:10:53.716 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/zonda_async_trades_update.py", line 28, in main
    '{str(dict_val['message']['transactions'][0]['id'])}',
KeyError: 'message'
21:13:11.185 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/zonda_async_trades_update.py", line 28, in main
    '{str(dict_val['message']['transactions'][0]['id'])}',
KeyError: 'message'
21:14:12.733 ERROR   $$ 'timestamp' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/zonda_async_trades_update.py", line 35, in main
    logger_conf().info(f"Trades received on timestamp: {dict_val['message']['timestamp']}")
KeyError: 'timestamp'
21:14:21.397 ERROR   $$ 'timestamp' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/zonda_async_trades_update.py", line 35, in main
    logger_conf().info(f"Trades received on timestamp: {dict_val['message']['timestamp']}")
KeyError: 'timestamp'
21:14:38.083 ERROR   $$ 'timestamp' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/zonda_async_trades_update.py", line 35, in main
    logger_conf().info(f"Trades received on timestamp: {dict_val['message']['timestamp']}")
KeyError: 'timestamp'
21:14:38.867 ERROR   $$ 'timestamp' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/zonda_async_trades_update.py", line 35, in main
    logger_conf().info(f"Trades received on timestamp: {dict_val['message']['timestamp']}")
KeyError: 'timestamp'
21:30:57.306 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:57.407 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:57.500 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:57.585 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:57.908 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:58.024 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:58.102 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:58.187 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:58.256 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:58.340 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:58.456 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:58.524 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:58.842 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:58.911 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:58.989 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:59.058 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:59.190 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:59.305 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:59.412 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:59.506 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:59.613 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:59.691 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:59.775 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:59.844 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:30:59.913 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:00.091 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:00.361 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:00.446 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:00.598 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:00.646 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:00.731 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:00.778 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:00.862 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:00.916 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:01.117 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:01.279 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:01.395 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:01.480 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:01.618 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:01.718 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:01.865 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:01.955 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:02.150 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:02.282 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:02.351 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:02.451 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:02.598 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:02.670 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:02.768 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:02.837 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:02.921 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:03.100 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:03.184 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:03.269 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:03.385 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:03.469 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:03.638 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:03.701 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:03.770 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:03.839 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:03.970 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:04.112 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:04.186 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:04.255 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:04.324 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:04.424 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:04.603 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:04.677 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:04.788 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:04.872 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:04.957 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:05.126 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:05.188 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:05.273 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:05.342 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:05.426 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:05.504 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:05.589 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:05.658 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:05.743 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:05.821 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:05.890 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:05.990 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:06.128 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:06.275 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:06.391 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:06.508 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:06.630 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:06.777 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:06.862 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:06.930 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:07.031 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:07.109 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:07.193 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:07.262 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:07.383 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:07.478 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:07.647 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:07.779 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:07.879 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:07.948 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:08.080 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:08.133 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:08.233 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:08.311 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:08.426 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:08.495 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:08.596 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:08.754 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:08.817 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:08.912 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:08.997 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:09.128 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:09.186 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:09.250 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:09.366 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:09.466 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:09.582 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:09.650 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:09.751 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:09.813 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:09.936 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:10.036 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:10.104 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:10.183 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:10.252 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:10.314 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:10.383 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:10.452 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:10.568 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:10.668 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:10.731 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:10.816 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:10.900 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:10.988 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:11.082 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:11.170 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:11.254 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:11.332 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:11.401 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:11.476 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:11.586 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:11.671 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:11.756 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:11.818 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:11.903 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:11.972 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:12.123 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:12.203 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:12.288 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:12.357 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:12.419 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:12.488 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:12.557 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:12.620 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:12.720 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:12.805 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:12.889 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:12.943 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:13.059 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:13.121 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:13.190 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:13.274 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:13.375 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:13.460 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:13.576 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:13.645 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:13.723 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:13.861 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:13.961 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:14.077 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:14.161 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:14.262 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:14.346 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:14.424 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:14.493 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:14.593 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:14.678 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:14.763 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:14.847 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:14.941 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:14.995 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:15.126 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:15.195 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:15.280 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:15.349 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:15.465 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:15.601 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:15.649 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:15.750 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:15.812 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:15.897 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:15.966 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:16.097 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:16.166 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:16.298 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:16.398 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:16.467 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:16.552 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:16.630 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:16.730 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:16.799 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:16.852 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:16.952 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:17.068 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:17.153 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:17.216 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:17.354 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:17.432 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:17.485 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:17.601 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:17.670 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:17.748 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:17.855 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:17.918 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:18.034 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:18.137 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:18.198 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:18.294 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:18.362 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:18.440 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:18.541 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:18.672 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:18.735 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:18.819 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:18.888 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:18.988 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:19.104 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:19.188 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:19.242 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:19.342 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:19.420 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:19.520 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:19.605 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:19.679 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:19.789 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:19.858 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:19.990 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:20.074 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:20.175 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:20.237 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:20.360 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:20.438 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:20.607 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:20.676 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:20.760 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:20.839 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:20.892 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:20.997 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:21.092 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:21.176 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:21.245 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:21.361 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:21.439 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:21.578 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:21.647 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:21.725 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:21.825 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:21.879 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:21.964 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:22.095 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:22.164 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:22.227 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:22.296 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:22.364 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:22.464 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:22.595 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:22.665 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:22.749 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:22.812 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:22.896 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:22.965 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:23.050 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:23.128 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:23.197 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:23.266 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:23.366 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:23.482 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:23.613 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:23.667 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:23.767 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:23.845 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:23.945 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:24.068 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:24.152 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:24.215 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:24.299 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:24.373 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:24.443 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:24.526 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:24.587 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:24.748 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:24.891 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:24.991 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:25.085 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:25.175 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:25.239 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:25.323 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:25.392 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:25.493 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:25.571 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:25.640 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:25.724 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:25.793 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:25.872 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:25.956 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:26.126 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:26.210 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:26.342 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:26.442 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:26.573 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:26.644 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:26.758 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:26.827 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:26.896 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:26.964 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:27.096 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:27.159 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:27.243 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:27.319 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:27.410 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:27.470 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:27.551 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:27.646 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:27.746 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:27.831 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:27.931 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:28.025 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:28.147 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:28.210 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:28.299 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:28.379 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:28.511 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:28.611 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:28.680 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:28.749 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:28.827 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:28.896 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:29.012 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:29.112 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:29.181 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:29.266 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:29.313 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:29.397 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:29.482 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:29.590 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:29.682 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:29.751 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:29.867 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:29.952 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:30.099 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:30.168 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:30.237 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:30.315 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:30.400 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:30.469 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:30.602 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:30.683 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:30.761 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:30.834 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:30.915 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:31.106 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:31.200 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:31.253 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:31.338 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:31.407 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:31.501 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:31.623 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:31.707 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:31.808 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:31.870 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:31.960 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:32.086 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:32.171 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:32.224 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:32.324 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:32.387 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:32.487 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:32.572 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:32.657 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:32.757 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:32.842 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:32.904 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:33.027 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:33.089 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:33.174 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:33.227 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:33.490 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:33.575 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:33.728 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:33.813 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:33.913 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:33.991 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:34.114 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:34.176 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:34.261 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:34.345 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:34.392 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:34.477 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:34.593 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:34.668 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:34.731 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:34.816 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:34.894 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:34.963 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:35.079 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:35.164 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:35.233 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:35.317 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:35.396 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:35.449 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:35.565 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:35.681 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:35.735 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:35.835 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:35.882 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:35.951 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:36.113 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:36.182 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:36.251 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:36.336 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:36.421 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:36.514 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:36.637 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:36.715 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:36.800 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:36.884 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:31:36.969 ERROR   $$ 'message' $$ 
Traceback (most recent call last):
  File "C:/Users/oskar/Desktop/algorytm/REST_API_on_demand_request.py", line 76, in main
    {int(dict_val['message']['timestamp'])}
KeyError: 'message'
21:32:41.533 INFO  Ob received on timestamp: 1642105961477
21:32:41.666 INFO  Ob received on timestamp: 1642105961653
21:32:41.744 INFO  Ob received on timestamp: 1642105961653
21:32:41.828 INFO  Ob received on timestamp: 1642105961653
21:32:41.913 INFO  Ob received on timestamp: 1642105961916
21:32:41.982 INFO  Ob received on timestamp: 1642105961916
21:32:42.348 INFO  Ob received on timestamp: 1642105961916
21:32:42.446 INFO  Ob received on timestamp: 1642105962441
21:32:42.568 INFO  Ob received on timestamp: 1642105962441
21:32:42.669 INFO  Ob received on timestamp: 1642105962657
21:32:42.769 INFO  Ob received on timestamp: 1642105962657
21:32:42.869 INFO  Ob received on timestamp: 1642105962856
21:32:42.969 INFO  Ob received on timestamp: 1642105962856
21:32:43.101 INFO  Ob received on timestamp: 1642105963093
21:32:43.170 INFO  Ob received on timestamp: 1642105963093
21:32:43.270 INFO  Ob received on timestamp: 1642105963256
21:32:43.348 INFO  Ob received on timestamp: 1642105963256
21:32:43.448 INFO  Ob received on timestamp: 1642105963441
21:32:43.586 INFO  Ob received on timestamp: 1642105963572
21:32:43.671 INFO  Ob received on timestamp: 1642105963572
21:32:43.771 INFO  Ob received on timestamp: 1642105963763
21:32:43.871 INFO  Ob received on timestamp: 1642105963763
21:32:43.972 INFO  Ob received on timestamp: 1642105963957
21:32:44.050 INFO  Ob received on timestamp: 1642105963957
21:32:44.135 INFO  Ob received on timestamp: 1642105964136
21:32:44.219 INFO  Ob received on timestamp: 1642105964136
21:32:44.304 INFO  Ob received on timestamp: 1642105964301
21:32:44.389 INFO  Ob received on timestamp: 1642105964301
21:32:44.489 INFO  Ob received on timestamp: 1642105964481
21:32:44.620 INFO  Ob received on timestamp: 1642105964610
21:32:44.805 INFO  Ob received on timestamp: 1642105964784
21:32:44.921 INFO  Ob received on timestamp: 1642105964907
21:32:45.074 INFO  Ob received on timestamp: 1642105965037
21:32:45.175 INFO  Ob received on timestamp: 1642105965160
21:32:45.253 INFO  Ob received on timestamp: 1642105965160
21:32:45.322 INFO  Ob received on timestamp: 1642105965160
21:32:45.407 INFO  Ob received on timestamp: 1642105965411
21:32:45.476 INFO  Ob received on timestamp: 1642105965411
21:32:45.654 INFO  Ob received on timestamp: 1642105965605
21:32:45.739 INFO  Ob received on timestamp: 1642105965742
21:32:45.808 INFO  Ob received on timestamp: 1642105965742
21:32:45.877 INFO  Ob received on timestamp: 1642105965742
21:32:45.955 INFO  Ob received on timestamp: 1642105965961
21:32:46.108 INFO  Ob received on timestamp: 1642105966100
21:32:46.193 INFO  Ob received on timestamp: 1642105966100
21:32:46.278 INFO  Ob received on timestamp: 1642105966278
21:32:46.356 INFO  Ob received on timestamp: 1642105966278
21:32:46.456 INFO  Ob received on timestamp: 1642105966442
21:32:46.594 INFO  Ob received on timestamp: 1642105966572
21:32:46.710 INFO  Ob received on timestamp: 1642105966709
21:32:46.778 INFO  Ob received on timestamp: 1642105966709
21:32:47.095 INFO  Ob received on timestamp: 1642105967050
21:32:47.179 INFO  Ob received on timestamp: 1642105967187
21:32:47.295 INFO  Ob received on timestamp: 1642105967187
21:32:47.380 INFO  Ob received on timestamp: 1642105967388
21:32:47.458 INFO  Ob received on timestamp: 1642105967388
21:32:47.597 INFO  Ob received on timestamp: 1642105967565
21:32:47.697 INFO  Ob received on timestamp: 1642105967689
21:32:47.782 INFO  Ob received on timestamp: 1642105967689
21:32:47.882 INFO  Ob received on timestamp: 1642105967874
21:32:47.960 INFO  Ob received on timestamp: 1642105967874
21:32:48.083 INFO  Ob received on timestamp: 1642105967874
21:32:48.161 INFO  Ob received on timestamp: 1642105968171
21:32:48.246 INFO  Ob received on timestamp: 1642105968171
21:32:48.346 INFO  Ob received on timestamp: 1642105968342
21:32:48.431 INFO  Ob received on timestamp: 1642105968342
21:32:48.585 INFO  Ob received on timestamp: 1642105968531
21:32:48.685 INFO  Ob received on timestamp: 1642105968673
21:32:48.764 INFO  Ob received on timestamp: 1642105968673
21:32:48.864 INFO  Ob received on timestamp: 1642105968856
21:32:48.948 INFO  Ob received on timestamp: 1642105968856
21:32:49.102 INFO  Ob received on timestamp: 1642105969066
21:32:49.202 INFO  Ob received on timestamp: 1642105969201
21:32:49.287 INFO  Ob received on timestamp: 1642105969201
21:32:49.387 INFO  Ob received on timestamp: 1642105969382
21:32:49.466 INFO  Ob received on timestamp: 1642105969382
21:32:49.587 INFO  Ob received on timestamp: 1642105969555
21:32:49.688 INFO  Ob received on timestamp: 1642105969669
21:32:49.766 INFO  Ob received on timestamp: 1642105969669
21:32:49.866 INFO  Ob received on timestamp: 1642105969853
21:32:49.935 INFO  Ob received on timestamp: 1642105969853
21:32:50.088 INFO  Ob received on timestamp: 1642105970023
21:32:50.166 INFO  Ob received on timestamp: 1642105970155
21:32:50.235 INFO  Ob received on timestamp: 1642105970155
21:32:50.336 INFO  Ob received on timestamp: 1642105970155
21:32:50.420 INFO  Ob received on timestamp: 1642105970395
21:32:50.489 INFO  Ob received on timestamp: 1642105970395
21:32:50.621 INFO  Ob received on timestamp: 1642105970613
21:32:50.705 INFO  Ob received on timestamp: 1642105970613
21:32:50.806 INFO  Ob received on timestamp: 1642105970800
21:32:50.890 INFO  Ob received on timestamp: 1642105970800
21:32:50.968 INFO  Ob received on timestamp: 1642105970800
21:32:51.107 INFO  Ob received on timestamp: 1642105971109
21:32:51.191 INFO  Ob received on timestamp: 1642105971109
